lldb/include/lldb/Utility/Stream.h

```diff
@@ -231,6 +231,18 @@ class Stream {
   ///     The string to be output to the stream.
   size_t PutCString(llvm::StringRef cstr);
 
+  /// Output a C string to the stream with color highlighting.
+  ///
+  /// Print a C string \a text to the stream, applying color highlighting to
+  /// the specified \a pattern within the string.
```

Comment:
Maybe:
Print a C string \a text to the stream, applying color highlighting to the portions of the string matched by the regex pattern \a pattern.

Just to emphasise the regex nature of this.

(someone may argue to call this parameter `re_pattern` as well, but don't change that now, only if later reviewers feel it's needed)

Reply:
> Print a C string \a text to the stream, applying color highlighting to the portions of the string matched by the regex pattern \a pattern.

Added the information and used the above as template